src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -25,6 +26,7 @@ export interface AccordionProps {
   showOpenAll?: boolean;
   onToggleAll?: (open: boolean) => void;
   onSectionOpen?: (accordionSection: { id: string; title: string }) => void;
+  visuallyHiddenText?: string;
```

Comment:
Would get a little more specific and name this `toggleAllHiddenText` as this could currently be referencing any text in this component. Note that we could drop the 'visually' part as just the word 'hidden' is sufficient to convey the meaning.